engine/src/test/java/io/zeebe/engine/util/StreamProcessorRule.java

```diff
@@ -181,6 +181,10 @@ public MutableZeebeState getZeebeState() {
     return streamProcessingComposite.getZeebeState();
   }
 
+  public long getLastProcessedPosition(final int partitionId) {
```

Comment:
I added this instead of reusing getLastSuccessfulProcessRecordPosition because, well, I'm not sure why that one is there in the first place :sweat_smile: If I read correctly, if you have multiple partitions, the zeebeState is that of the last started one? I think it's a confusing API imo, but I also didn't want to refactor everything.